src/unpkg.mjs

```diff
@@ -38,7 +38,7 @@ async function transformBody(resp, req) {
 
 async function cleanupResponse(resp, req) {
   try {
-    return transformBody(cleanupHeaders(resp), req);
+    return await transformBody(cleanupHeaders(resp), req);
```

Comment:
@trieloff I added an await here otherwise the catch and fallback would never be hit. Since this function is async I think that was the intention, right?

Reply:
You don't need to `await` a `return` value. The `return` value of an `async` function is always a `Promise` and Promises within promises get auto-unwrapped.

Reply:
Yes, however if the code inside `transformBody(cleanupHeaders(resp), req);` throws an exception it never ends up in the catch block on the next line. I think it gets caught where `cleanupResponse()` is awaited in that case. The intended fallback (just calling `cleanupHeaders()` ) is not called then.
Adding the await here will make the fallback part of this method work.